### DISCUSSION TITLE
Processing Burst Data for InSAR

### DISCUSSION THREAD

**Discussion by pbrotoisworo** [ASFHyP3/burst2safe#34] [Q&A] [answered]
Hello,

This package seems like a great opportunity to reduce processing time for InSAR processing. Would this follow the standard Sentinel-1 SLC processing methodology for InSAR in SNAP? Something like

1. Apply Orbit File for image 1 and image 2
2. Back-Geocoding
3. Inteferogram
4. TOPSAR-Deburst
5. etc...

**Comment by forrestfwilliams** [comment, marked as answer]
Hey @pbrotoisworo I'm less familiar with the processing workflow in SNAP, but yes. The goal of burst2safe is to provide you with a SAFE file that:

1.  Is compatible with the majority of existing S1 processors.
2. Contains only the bursts you actually want to process.

You should be able to use the SAFEs created by burst2safe just like any other SAFE file that you'd download from ESA or ASF. If you run into any issues using burst2safe-generated SAFEs in SNAP, let me know! We can work on fixing any issues that arise.
